docs/source/internals.rst

```diff
@@ -23,3 +23,42 @@ open an issue in our issue tracker, JIRA:
    the issue and the steps to reproduce it.
 
 Bug reports in JIRA for this project can be viewed by everyone.
+
+.. _supported-versions-policy:
+
+Supported versions
+==================
+
+Django MongoDB Backend follows Django's :ref:`supported versions policy
+<django:supported-versions-policy>`.
+
+The main development branch of Django MongoDB Backend follows the most recent
+:term:`django:feature release` of Django and gets all new features and
+non-critical bug fixes.
+
+Security fixes and data loss bugs will also be applied to the previous feature
+release branch, and any other supported long-term support release branches.
+
```

Comment:
Maybe a sub-header "Example" here

Reply:
This is copied from Django's documentation. A rule of thumb is to think of headings like an outline and to try to avoid a single subheading.